src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -1,57 +1,97 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router';
+import { bindActionCreators } from 'redux';
 import * as DS from '@nypl/design-system-react-components';
+import { getQueryString } from '../../search/query';
+import * as searchActions from '../../actions/SearchActions';
 
-/**
- * Takes `query` and `type` as properties to pass to its methods.
- *
- * @param {object} props
- * @returns {array}
- */
-const Breadcrumbs = ({ links, pageType, onClickHandler }) => {
-  const crumbTrail = () => {
-    const homeLink = (
-      <Link
-        to="/"
-        onClick={event => onClickHandler(event)}
-      >
-        ResearchNow
-      </Link>
-    );
-
-    const crumbs = [homeLink];
-
-    if (links && links.length && pageType !== 'home') {
-      // If not on home, add all the other links, except for the current page
-      const additionalLinks = (links.slice(0, -1)
-        .map(link => (
-          <Link to={link.href}>
-            {link.text}
-          </Link>
-        )));
-      if (additionalLinks) {
-        crumbs.concat(additionalLinks);
-      }
-    }
-    return crumbs;
-  };
+export const getBreadcrumbLinks = (searchQuery, workDetail) => {
```

Comment:
I like having this method be very explicit about what to output for each page type but I worry about what happens as we add more page types. There aren't that many more (maybe 2?) in the roadmap but if this has to handle any more than that I think this might get unwieldy.

This is definitely a "future-proofing" concern and not something that should block this fix. I think it could be handled pretty easily down the road with an object representing the expected behavior for each page type.